industrial_ci/src/tests/source_tests.sh

```diff
@@ -149,6 +149,13 @@ set +o pipefail
 
 ici_time_end  # rosdep_install
 
+if [ "$CATKIN_LINT" == "true" ]; then
+    ici_time_start catkin_lint
+    sudo apt-get install -qq -y python-catkin-lint || error
```

Comment:
`|| error` is not needed, the script will fail on all errors.
Please use the pip version for compatibility with Debian.